app/models/operatingsystems/redhat.rb

```diff
@@ -16,6 +16,15 @@ def mediumpath(medium_provider)
     end
   end
 
+  def additional_media(medium_provider)
+    additional_media = []
+    medium_provider.additional_media.each do |medium|
+      next unless medium && medium[:name] && medium[:url]
+      additional_media << "repo --name \"#{medium[:name]}\" --baseurl \"#{medium[:url]}\"#{medium[:install] ? ' --install' : ''}"
```

Comment:
I'm wondering if we should do this kind of rendering in the template. That way this is more flexible.